desktop/release-notes.md

```diff
@@ -24,12 +24,118 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.12.0
+2022-09-01
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+## New
+
+- Added the experimental docker daemon with the containerd image store.
+- Allow Docker Desktop to run untagged images. Fixes [https://github.com/docker/for-mac/issues/6425]
+- Add search capabilities to Docker Extension Marketplace. Fixes [docker/roadmap#346](https://github.com/docker/roadmap/issues/346).
+- Copied terminal contents do not contain non-breaking spaces anymore.
+- Enabled Compose V2 by default on new installations of Docker Desktop.
+- Added View menu with Zoom In, Zoom Out, and Actual Size options to control the size of the Docker Desktop UI.
```

Comment:
Maybe i missed something, but i can't see this feature in DD 🤔

Reply:
On mac there's a new View menu next to Docker Desktop and Edit at the top, with the available actions mentioned here. I assume this shows up on win and linux in a similar fashion: ![Screenshot 2022-08-31 at 23 08 02](https://user-images.githubusercontent.com/54431776/187773486-bfbaee1a-8520-4232-ad83-050d45a85269.png)

Reply:
Ah ok, I have Windows so can't see this. Think this should be moved to the Mac section in 'Bugs and minor changes'

Reply:
The Zoom In, Zoom Out, and Actual Size also works for Windows with the keyboard shortcuts CTRL +/CTRL -/CTRL 0. So the feature itself works for all, but Windows Docker Dashboard doesn't have any menu like on Mac.
Maybe we should rephrase it a bit, because it's still an useful feature on Windows as well.

Reply:
How about:

Added the ability to zoom in, out or set Docker Desktop to Actual Size. This is through the **View** menu on Mac or using keyboard shortcuts `CTRL +`, `CTRL -`, `CTRL 0` on Windows

Reply:
Minor suggestion to include shortcuts on mac as well: Added the ability to zoom in, out or set Docker Desktop to Actual Size. This is done by using keyboard shortcuts `⌘ +` / `CTRL +`, `⌘ -` / `CTRL -`, `⌘ 0` / `CTRL 0` on Mac and Windows respectively, or through the View menu on Mac.

Reply:
Perfect 👍